src/simulator_core/simulation/networksimulation.py

```diff
@@ -13,21 +13,37 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-""" Simulates an heat network for the specified duration."""
+"""Simulates an heat network for the specified duration."""
 
 from simulator_core.entities import HeatNetwork, NetworkController, SimulationConfiguration
+import logging
+
+logger = logging.getLogger(__name__)
 
 
 class NetworkSimulation:
-    """NetworkSimulation connects the controller and HeatNetwork (incl. assets). """
+    """NetworkSimulation connects the controller and HeatNetwork (incl. assets)."""
+
     def __init__(self, network: HeatNetwork, controller: NetworkController):
-        """Instantiate the NetworkSimulation object"""
+        """Instantiate the NetworkSimulation object."""
         self.config = None
         self.network = network
         self.controller = controller
 
     def run(self, config: SimulationConfiguration):
         """Run the simulation.
 
-        :param SimulationConfiguration"""
-        self.config = config
+        :param SimulationConfiguration config: Configuration to run the simulation with.
+        """
+        self.config = config  # Why do we store it in here?
```

Comment:
Good question? Why do we need to carry the config?

Reply:
I will remove it